utils/python/create_test_impl.py

```diff
@@ -347,7 +352,11 @@ def _build_phase(self, test_name):
     def _run_phase(self, test_name):
     ###########################################################################
         test_dir = self._get_test_dir(test_name)
-        return self._run_phase_command(test_name, "./case.submit", RUN_PHASE, from_dir=test_dir)
+        # TODO: Submit should work in either case.
+        if (self._no_batch):
```

Comment:
We can't not use submit since that is the mechanism via which archiving is handled.   The
ERR test in cime_developer tests this feature.